app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ItemFunction.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.warehousemanagement.domain.feature.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.warehousemanagement.R
+
+
+
+@Composable
+fun ItemFunction(functionName: String, iconResource: Int, size: Dp) {
```

Comment:
What happen when I click into this Itemfunction? Nothing !!

Reply:
done